tmt/steps/provision/mrack.py

```diff
@@ -91,8 +92,13 @@ def import_and_load_mrack_deps(workdir: Any, name: str) -> None:
         # HAX remove mrack stdout and move the logfile to /tmp
         mrack.logger.removeHandler(mrack.console_handler)
         mrack.logger.removeHandler(mrack.file_handler)
-        if os.stat("mrack.log"):
+
+        try:
             os.remove("mrack.log")
+        except OSError as e:
+            if e.errno != errno.ENOENT:  # errno.ENOENT = no such file or directory
+                ProvisionError(str(e))
```

Comment:
A more descriptive message would be better, to provide higher level of context to the user, plus `… from e` to preserve the original exception.

Reply:
Warning sounds also good. If it's not critical, it's not necessary to kill the whole run at the finish line because of a missing log.

Reply:
I hope this version is more satisfying. @dav-pascual please have a look as well.

Reply:
pre-commit complains about unused import however except that it looks good.

Reply:
Shouldn't there supposed to be a warning? In the most recent version, I see an explicit `pass`, effectively silencing the exception. Is that correct?

Reply:
If I understand @Tiboris correctly, this is just a conditional cleanup of the log file which just might not be present (and it's not a problem). So I'd say don't want to bother user with any warning, right? Or did I get it wrong?

Reply:
Yes correct let me rephrase commit